<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpListener</class>
 <widget class="QWidget" name="UdpListener">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>484</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UdpListener</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="ipAddress">
         <property name="text">
          <string extracomment="IP address">239.255.43.21</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="port">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>45454</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bind">
         <property name="text">
          <string>Bind</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="UdpTableView" name="udpTableView">
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>300</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedDiscoveryView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="instantBitRate">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="averageBitRate">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="maxBitRate">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>UdpTableView</class>
   <extends>QTableView</extends>
   <header>../UDP-Listener/udptableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ipAddress</sender>
   <signal>returnPressed()</signal>
   <receiver>bind</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>747</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
